python/recordio/recordio.py

```diff
@@ -22,12 +22,18 @@ class writer(object):
     writer is a recordio writer.
     """
 
-    def __init__(self, path, maxChunkSize = -1, compressor = -1):
+    def __init__(self, path, maxChunkSize=-1, compressor=-1, logger=None):
```

Comment:
Can you help to check `maxChunkSize` to `max_chunk_size` ?

Reply:
Another question: is this change only for debug? Or is there any other background that necessitates this change?

Reply:
yeah, only to add more log to facilitate debugging